libs/ui/src/hooks/ui_strings_api.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryClient, QueryKey, useQuery } from '@tanstack/react-query';
+import { UiStringsApi } from '@votingworks/types';
+import * as grout from '@votingworks/grout';
+
+// Unable to use `grout.Client<UseStringsApi>` directly due to some mismatched
+// type inference with `grout.AnyMethods`, so copying the `grout.Client`
+// definition here for now:
```

Comment:
Seems like a reasonable workaround!

I think `grout.Client<UseStringsApi>` in the comment is maybe supposed to read `grout.Client<UiStringsApi>`

Reply:
oops - thanks!